elastic/apm-server
# Remove `ecs.version` field from mappings

## Motivation/summary

Remove the `ecs.version` field mapping from APM data streams. We don't use this field at all in APM UI, and its presence works against converging on OTel SemConv, and decoupling the index templates from apm-server.

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. Ingest some APM data (e.g. with `apmtool`)
2. Verify the `ecs.version` field is not present in any APM docs
3. Verify basic APM UI functionality still works

## Related issues

https://github.com/elastic/elasticsearch/pull/97546
